config/application.rb

```diff
@@ -2,6 +2,7 @@
 
 require "rails/all"
 
+config.assets.paths << Rails.root.join('app', 'assets', 'fonts')
```

Comment:
@qiyundai  This line is crashing the app
https://lewagon-alumni.slack.com/archives/C033YB2VC/p1618579293014200

Reply:
The thread mentions the line could be placed elsewhere. Just not sure if this is the right approche*